docs/modeling/triggers.md

```diff
@@ -105,7 +118,57 @@ Actions are stored as a `payload`, for example using the send a signal action fo
 			"subject": "${emailSubject}"
 		},
 		"name": "Signal_0n91cib"
+	}
+}
+```
+
+#### Receive a message
+The action to receive a message will send a named [message](../modeling/processes/bpmn/message.md) when an event criteria are met. The payload for a receive message action is:
 
+| Property | Description | Example | Required | 
+| -------- | ----------- | ------- | -------- |
+| `Name` | The name of the message to send. There must be an [intermediate message catching event](../modeling/processes/bpmn/message.md#message-intermediate-catch-events) or [message boundary event](../modeling/processes/bpmn/boundary.md#message-boundary-events) with the same name for the payload to be received. | Message_077epax | Yes | 
+| `correlationKey` | An optional [correlation key](../modeling/processes/bpmn/message.md#correlation-keys) can be provided for matching the message | 014-245 | No | 
+| `payloadType` | The payload type is set automatically to `ReceiveMessagePayload`. | | Yes |
+| `variables` | Values from the trigger can be mapped to [process variables](../modeling/processes/variables.md). For a connector this will include the output parameters configured in the [connector event](../modeling/connectors/README.md#events) definition. As a signal can be caught by multiple catching events, the variables must be written in JSON format. | | No | 
```

Comment:
I would avoid talking about signals here as we are describing messages. We have to write this in JSON format because message is not linked to a process definition but to a name so there can be more than one process definition using the same signal name. This makes us impossible to know which are the variables available for the message to be mapped

Reply:
Same in the start message action

Reply:
Ah yes, that makes sense thanks.